src/app/page.tsx

```diff
@@ -1,317 +1,186 @@
+            className="animate-spin h-5 w-5 mr-3"
+          >
+            <path
+              d="M12,1A11,11,0,1,0,23,12,11,11,0,0,0,12,1Zm0,19a8,8,0,1,1,8-8A8,8,0,0,1,12,20Z"
+              opacity=".25"
             />
-            {"Hide Disqualified"}
-          </div>
-          {activeTab === "individualMastersSongStandings" && (
-            <div style={{ display: "flex", gap: "8px" }}>
-              <Switch
-                onChange={setShouldHideFinalists}
-                checked={shouldHideFinalists}
-              />
-              {"Hide Finalists"}
-            </div>
-          )}
+            <path d="M10.14,1.16a11,11,0,0,0-9,8.92A1.59,1.59,0,0,0,2.46,12,1.52,1.52,0,0,0,4.11,10.7a8,8,0,0,1,6.66-6.61A1.42,1.42,0,0,0,12,2.69h0A1.57,1.57,0,0,0,10.14,1.16Z" />
+          </svg>
+          Loading...
         </div>
-        <Button
-          icon={<RedoOutlined />}
-          type="default"
-          disabled={isFetchingStandings}
-          onClick={() => {
-            if (isLeaderboardFrozen) {
-              api.info({
-                message: "Bruh, calm down",
-                description:
-                  "The leaderboard is still frozen. I promise it'll unfreeze when the qualifiers are over, which should happen soon. Be patient, okay? :›",
-                placement: "bottomRight",
-                duration: 6,
-              });
-            }
-            fetchStandings();
-          }}
-        >
-          Refresh
-        </Button>
-      </div>
-      {isLeaderboardFrozen && (
-        <div
-          style={{
-            marginTop: "12px",
-            marginBottom: "4px",
-            fontWeight: "bold",
-            fontSize: "large",
-            textAlign: "center",
-          }}
-        >
-          The leaderboard is currently frozen
+      )}
+      {data?.qualifiers.length === 0 && (
+        <div className="flex min-h-max justify-center items-center mt-8">
+          No submissions yet.
         </div>
       )}
-      <Tabs
-        style={{ marginTop: "8px" }}
-        onChange={setActiveTab}
-        defaultActiveKey="masters"
-        items={[
-          {
-            key: "masters",
-            label: "Masters",
-            children: table(masterSongs, masterStandings, "masters", 8),
-          },
-          {
-            key: "individualMastersSongStandings",
-            label: "Individual Songs (Masters)",
-            children: (
-              <IndividualSongLeaderboard
-                songs={individualMastersSongs}
-                loading={isFetchingStandings}
-                standings={individualSongStandings}
-                options={{
-                  shouldHideFinalists,
-                  shouldHideDisqualified,
-                }}
-              />
-            ),
-          },
-        ]}
-      />
+      {/* </TabsContent>
+        <TabsContent value="individuals">
+          Change your password here.
+        </TabsContent>
+      </Tabs> */}
```

Comment:
Wait a second, what does this do?

Is `Change your password here` just some placeholder flavour text?

Reply:
Yup, it is

_For now_